How-to-setup-a-bitcoin-node-on-Synology-NAS/index.md

```diff
@@ -36,20 +36,20 @@ rpcuser=yourusername
 rpcpassword=yourpassword
 ```
 
-Next, we will need to install docker by accessing the `Package Center` from our NAS administration page. From here search for the `docker package`, then select the and install option.
+Next, we will need to install docker by accessing the `Package Center` from our NAS administration page. From here search for the `docker package`, then select the and `install` option.
 
 ![](images/nRDd07S.png)
 
- We are now ready to install our bitcoin docker instance by visiting opening the `Docker` package, then selecting "Registry" and searching for "bitcoind". I recommend downloading the `kylemanna` image which will simply install bitcoind on an Ubuntu instance and from its official repository - feel free to inspect the code at [Github](https://github.com/kylemanna/docker-bitcoind/blob/master/Dockerfile)
+ We are now ready to install our bitcoin docker container by opening the `Docker` package, then selecting `Registry` and searching for "bitcoind". I recommend downloading the `kylemanna` image which will simply install bitcoind on an Ubuntu instance from its official repository - feel free to inspect the code at [Github](https://github.com/kylemanna/docker-bitcoind/blob/master/Dockerfile)
 
 ![](images/wiZtXhR.png)
 
-Next, go to `Images` within Docker and select the `Launch` option of the image you just downloaded. A `Create Container` dialog is shown. You can adjust the name to something which suits your needs. We can leave the other options at default unless you have any specific preferences here. Next, select the `Advanced Settings` option.
+Next, go to `Images` within Docker and select the `Launch` option of the image you just downloaded. A `Create Container` dialog is shown. You can adjust the name to something which suits your needs e.g. bitcoind. We can leave the other options at default unless you have any specific preferences here. Next, select the `Advanced Settings` option.
 
 ![](images/gmtcZSJ.png)
 
-To restart your container after an OS update or restart, select the `Enable auto-restart`.
-From the  `Volume`, select `Add Folder`, enter your local path in `File/Folder` and `/bitcoin/.bitcoin` as `Mount path`.
+To restart your container after an OS update or reboot, select the `Enable auto-restart`.
+From the `Volume` tab, select `Add Folder`, enter your local path in `File/Folder` and `/bitcoin/.bitcoin` as `Mount path`.
```

Comment:
Are these correct as per the instruction @roshii ?  The tutorial suggests were setting up a bitcoin container in testnet mode. Should we update this screen to reflect the aforementioned bitcoin-testnet folder instead?

Reply:
this is correct yes, local path = `bitcoin-testnet` bit I should make this more clear